phpunit.xml.dist

```diff
@@ -22,14 +22,16 @@
             <directory>./</directory>
             <exclude>
                 <directory>./Tests/</directory>
+                <directory>./DataFixtures/</directory>
                 <directory>./Resources/</directory>
-                <directory>./DependencyInjection/</directory>
                 <directory>./vendor/</directory>
-
-                <!-- Ignore ODM driver -->
-                <directory>./Document/</directory>
+                <directory>./coverage/</directory>
             </exclude>
         </whitelist>
     </filter>
 
+    <php>
+        <ini name="precision" value="8"/>
+    </php>
```

Comment:
This does not match your commit message.

Reply:
Not sure what it does, I am trying to make all phpunit configs look the same, copied from SonataAdminBundle